server/bookings/model.ts

```diff
@@ -5,10 +5,14 @@ export type CreateBooking = {
   user: string;
   date: string;
   office: string;
+  parking: boolean;
 };
 
 export const isCreateBooking = (arg: any): arg is CreateBooking =>
-  typeof arg.user === 'string' && typeof arg.date === 'string' && typeof arg.office === 'string';
+  typeof arg.user === 'string' &&
+  typeof arg.date === 'string' &&
+  typeof arg.office === 'string' &&
+  typeof arg.parking === 'boolean';
```

Comment:
On second thoughts ... adding this breaks the current client implementation - we should make this optional, at least for now and default it to false.